hackathon_site/hardware/test_api.py

```diff
@@ -355,8 +357,21 @@ def setUp(self):
         OrderItem.objects.create(
             order=self.order_2, hardware=self.other_hardware,
         )
+        self.order_3 = Order.objects.create(
+            created_at=(datetime.now() + timedelta(days=1)).date(),
```

Comment:
We specify `auto_now_add=True` on this field in the model, which means the created at timestamp is generated by the database. It's actually not even possible to override it like this, so you can remove this line and the datetime import.

Reply:
how do I test the created_at ordering filter?

Reply:
To start, you know that the timestamps will be increasing in the order you created the orders in the code, so that's your target ordering. 

In the test, you can do `self._build_filter_url(ordering="created_at")` for ascending (so your order ids should be [1, 2, 3]), and `ordering="-created_at"` for descending.

Reply:
How accurate is the DateTimeField because I printed the response and it seems the orders are created at the exact same time I think

Reply:
Each one should be a separate database operation, so they should differ ever so slightly even if it's not printing a difference. Does the ordering not behave as expected?

Reply:
oh nvm it works 👍